internal/webhooks/certs.go

```diff
@@ -191,16 +191,11 @@ func GetWebhookServiceDNSNames(ctx context.Context, clientSet *kubernetes.Client
 	service := servicesList.Items[0]
 	serviceName := service.GetName()
 	serviceNamespace := service.GetNamespace()
-	serviceCluster := service.GetClusterName()
```

Comment:
In a future release of K8s client, this is actually depreacted with the message that it's not being used internally and was renamed to ZZZ_DeprecatedGetClusterName. I've premptively removed it and simply appended cluster.local to the dnsNames slice